src/test/java/org/folio/consortia/service/TenantServiceTest.java

```diff
@@ -157,6 +161,7 @@ void shouldSaveTenantWithExistingAndPermissions() {
     when(tenantRepository.findCentralTenant()).thenReturn(Optional.of(centralTenant));
     when(tenantRepository.save(any(TenantEntity.class))).thenReturn(tenantEntity1);
     doNothing().when(configurationClient).saveConfiguration(createConsortiaConfiguration(CENTRAL_TENANT_ID));
+    doNothing().when(userTenantsClient).postUserTenant(any());
     when(conversionService.convert(tenantEntity1, Tenant.class)).thenReturn(tenant);
     doReturn(folioExecutionContext).when(contextHelper).getSystemUserFolioExecutionContext(anyString());
     when(folioExecutionContext.getTenantId()).thenReturn("diku");
```

Comment:
please add verify() into tests that both configurationClient and userTenantsClient invoked, because not this test not checking it

Reply:
fixed